src/pages/api/graphql.tsx

```diff
@@ -92,25 +93,37 @@ export const apolloServer = new ApolloServer({
         userId: userId,
       }
 
-      const updateDocument = {
-        $set: {
-          displayName: displayName,
-        },
-      }
-
-      if (foundUser && !foundUser.displayName) {
-        await db.collection('users').updateOne(query, updateDocument)
-      }
-
       if (!foundUser) {
         try {
           const initCollection = await getExampleCollection()
-          await User.createOne(userId, [initCollection], displayName, { db })
+          await User.createOne(userId, [initCollection], displayName, 'light', {
+            db,
+          })
         } catch (e) {
           // TODO log error
           // console.error('error in creating new user', e)
           throw new ApolloError('Error creating new user')
         }
+      } else {
+        // TODO we should be able to remove these once all exisitng users have the defaults set.
```

Comment:
question/thought for later: how will be know when this happens?

Reply:
I guess we would just have to do this periodically, but maybe we could write a db query looking for any user that doesn't have the defaults set? Then at some point in the future, the query won't return anything and we'll know? There's probably an easier way, but that's just the first thought I had.

Reply:
yeah I think we'd have to run queries against the data, or think of an approach that was like a db migration where on deploy it looped through all our users and added the defaults if they are missing and would only be run once. Things are simple enough I'm okay if we just manually run a query at some point in the future and once it shows everyone has the defaults we can drop the code. 

There might be other ways though and if either of you dig into it or want to lets get it on our list for tech debt.

Reply:
I added a story in shortcut about this discussion sc-1128